src/utils/update/managing.officer.mapper.ts

```diff
@@ -18,7 +19,8 @@ export const mapToManagingOfficer = (officer: CompanyOfficer): ManagingOfficerIn
     has_former_names: officer.formerNames ? yesNoResponse.Yes : yesNoResponse.No,
     former_names: formernames,
     date_of_birth: mapDateOfBirth(officer.dateOfBirth),
-    nationality: officer.nationality,
+    nationality: nationalities[0],
+    second_nationality: nationalities[1],
```

Comment:
Maybe be explicit about the case when there is only one element in list
second_nationality: nationalities.length < 2? undefined: nationalities[2]

Reply:
if nationalities[1] doesn't exist then it will return undefined anyway, so this isn't necessary

Reply:
Yes, but might help to be explicit. Can also be ["",""] so we have undefined or "" being no second nationality.